### PR TITLE
add arize-phoenix as a package

### PR DESCRIPTION
**Pull request by mikeldking**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mikeldking** [comment]
@conda-forge/help-python ready for review!

**Comment by RogerHYang** [comment]
Yes, I am willing to maintain this package.

**Comment by fjcasti1** [comment]
Willing to maintain this package!

**Comment by mikeldking** [comment]
I am willing to maintain this package!

**Comment by mikeldking** [comment]
@conda-forge/help-python we use `jupyter-builder` to build our web resources but it's failing in CI. any help would be much appreciated!

File: recipes/arize-phoenix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.11
+    - hatchling
```
Comment (dopplershift) [inline recipes/arize-phoenix/meta.yaml:20]: You need to explicitly depend upon `hatch-jupyter-builder` I think:
```suggestion
    - hatchling
    - hatch-jupyter-builder
```

Comment (mikeldking) [reply recipes/arize-phoenix/meta.yaml:20]: Ah thank you so much @dopplershift

**Comment by mikeldking** [comment]
@conda-forge/help-python ready for review!

File: recipes/arize-phoenix/LICENSE
Comment (dopplershift) [inline recipes/arize-phoenix/LICENSE:1]: The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Recipe looks good. Just remove the unneeded file.

File: recipes/arize-phoenix/LICENSE
Comment (mikeldking) [reply recipes/arize-phoenix/LICENSE:1]: gotcha, will do!

**Comment by mikeldking** [comment]
@dopplershift @conda-forge/help-python - license removed (since included in the tarball)

Ready for review!

File: recipes/arize-phoenix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arize_phoenix-{{ version }}.tar.gz
+  sha256: c70febecd9618df8c850e9bf5a886b2d21594a7569b7361054ef4b112a8850c6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --install-option="--skip-npm" . -vv
```
Comment (mikeldking) [inline recipes/arize-phoenix/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

**Comment by mikeldking** [comment]
@dopplershift sorry for the bad param. Ready for real this time

**Pull request merged**